python/ansible/plugins/action/adcm_check.py

```diff
@@ -89,22 +130,46 @@ def run(self, tmp=None, task_vars=None):
                         "_msg are mandatory args of adcm_check")
             }
 
-        result = super(ActionModule, self).run(tmp, task_vars)
         title = self._task.args['title']
         result = self._task.args['result']
         msg = self._task.args.get('msg', '')
         fail_msg = self._task.args.get('fail_msg', '')
         success_msg = self._task.args.get('success_msg', '')
 
-        log.debug('ansible adcm_check: %s, %s, %s, %s, %s, %s',
-                  job_id, title, result, msg, fail_msg, success_msg)
+        group_title = self._task.args.get('group_title', '')
+        group_fail_msg = self._task.args.get('group_fail_msg', '')
+        group_success_msg = self._task.args.get('group_success_msg', '')
+
+        log_data = {
+            'title': title,
+            'result': result,
+            'msg': msg,
+            'fail_msg': fail_msg,
+            'success_msg': success_msg,
+            'group_title': group_title,
+            'group_success_msg': group_success_msg,
+            'group_fail_msg': group_fail_msg
+        }
+
+        log.debug('ansible adcm_check: %s',
+                  ', '.join([f'{k}: {v}' for k, v in log_data.items() if v]))
 
```

Comment:
Lets move all this group logic into cm.job.log_check(). Lets call create_group_log() and log_group_check() from log_check()

If it requires too many params for log_check(), you could group them to some dict/dicts

Reply:
Ok